test/extended/cli/adm_upgrade/recommend.go

```diff
@@ -305,3 +337,43 @@ python3 -m http.server --bind ::
 		Path:   "graph",
 	}, nil
 }
+
+func getDefaultIngressCertificate(ctx context.Context, oc *exutil.CLI) (string, error) {
```

Comment:
This is amazing!
I will try to remember the existence of these powerful functions in case I need them in the future.